1_fetch/src/data_extraction_ms.R

```diff
@@ -6,31 +6,29 @@
 ################################
 
 
-## Libs
-library(dplyr)
-library(readr)
-library(sbtools)
+## Libs - Commented out for targets
+# library(dplyr)
+# library(readr)
+# library(sbtools)
 
+## Check that you are in project home directory - should be in project home directory - Commented out for Targets 
+# print(getwd())
 
-## Check that you are in project home directory - should be in project home directory
-print(getwd())
-
-## Global Variables
-
-out_path <- '1_fetch/out'
-csv_filename_output <- 'model_RMSEs.csv'
-sb_id_sciencebase <- '5d925066e4b0c4f70d0d0599'
-
+## Global Variables - Commented out for targets
+# downloaded_data_folder <- '1_fetch/out'
+# file_name <- 'model_RMSEs.csv'
+# sb_id_sciencebase <- '5d925066e4b0c4f70d0d0599'
```

Comment:
Now that you're not using this commented-out code anymore, can you please remove it?